### ISSUE TITLE
NullPointerException opening database after server hungup. 

### ISSUE THREAD

**Issue by NicolasJuanF** [orientechnologies/orientdb#9658]
## Expected behavior  
Opening the database successfully. 

## Actual behavior  
When trying to open a database from Studio I get the message "Session expired" and the following log: 

```
2021-08-24 13:33:11:926 INFO  DWL:DB_NAME: block size = 4096 bytes, maximum segment size = 1476 MB [DoubleWriteLogGL]Exception `32090AC1` in storage `plocal:/opt/orientdb-3.2/databases/DB_NAME`: 3.2.0 (build c08b76230798b477a8abbf2312b30ecddad842ad, branch develop)
java.lang.NullPointerException
	at com.orientechnologies.orient.core.storage.cache.chm.AsyncReadCache.releaseFromRead(AsyncReadCache.java:310)
	at com.orientechnologies.orient.core.storage.impl.local.paginated.base.ODurableComponent.releasePageFromRead(ODurableComponent.java:179)
	at com.orientechnologies.orient.core.storage.index.sbtreebonsai.global.btree.BTree.firstItem(BTree.java:311)
	at com.orientechnologies.orient.core.storage.index.sbtreebonsai.global.btree.BTree.firstKey(BTree.java:230)
	at com.orientechnologies.orient.core.storage.ridbag.sbtree.OSBTreeCollectionManagerShared.load(OSBTreeCollectionManagerShared.java:87)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.open(OAbstractPaginatedStorage.java:423)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.getAndOpenStorage(OrientDBEmbedded.java:543)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:494)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:216)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:434)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:201)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:855)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:848)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.authenticate(OServerCommandAuthenticatedDbAbstract.java:194)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.beforeExecute(OServerCommandAuthenticatedDbAbstract.java:136)
	at com.orientechnologies.orient.server.network.protocol.http.command.get.OServerCommandGetConnect.beforeExecute(OServerCommandGetConnect.java:55)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:239)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:800)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:67)
Internal server error:

java.lang.NullPointerException
	at com.orientechnologies.orient.core.storage.cache.chm.AsyncReadCache.releaseFromRead(AsyncReadCache.java:310)
	at com.orientechnologies.orient.core.storage.impl.local.paginated.base.ODurableComponent.releasePageFromRead(ODurableComponent.java:179)
	at com.orientechnologies.orient.core.storage.index.sbtreebonsai.global.btree.BTree.firstItem(BTree.java:311)
	at com.orientechnologies.orient.core.storage.index.sbtreebonsai.global.btree.BTree.firstKey(BTree.java:230)
	at com.orientechnologies.orient.core.storage.ridbag.sbtree.OSBTreeCollectionManagerShared.load(OSBTreeCollectionManagerShared.java:87)
	at com.orientechnologies.orient.core.storage.impl.local.OAbstractPaginatedStorage.open(OAbstractPaginatedStorage.java:423)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.getAndOpenStorage(OrientDBEmbedded.java:543)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:494)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:216)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.open(OrientDBEmbedded.java:434)
	at com.orientechnologies.orient.core.db.OrientDBDistributed.open(OrientDBDistributed.java:201)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:855)
	at com.orientechnologies.orient.server.OServer.openDatabase(OServer.java:848)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.authenticate(OServerCommandAuthenticatedDbAbstract.java:194)
	at com.orientechnologies.orient.server.network.protocol.http.command.OServerCommandAuthenticatedDbAbstract.beforeExecute(OServerCommandAuthenticatedDbAbstract.java:136)
	at com.orientechnologies.orient.server.network.protocol.http.command.get.OServerCommandGetConnect.beforeExecute(OServerCommandGetConnect.java:55)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:239)
	at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:800)
	at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:67)
```

## Steps to reproduce  
I was using the db when my computer froze. After restarting the server no errors are logged only after trying to open the database I get the logs mentioned previously.

**Comment by andrii0lomakin** [comment]
Hi, that is a critical thing but maybe already fixed in another issue, could you send me server logs and the database itself for investigation?

**Comment by andrii0lomakin** [comment]
Fixed

**Comment by qianSir01** [comment]
hi,Can you tell me how you solved this problem?
